src/core/settings.js

```diff
@@ -5,10 +5,30 @@
  */
 import * as signal from "../core/signal-bus.js";
 import { gmi } from "./gmi/gmi.js";
+import fp from "../../lib/lodash/fp/fp.js";
 
 export const settingsChannel = "genie-settings";
 
+let game;
+
 export const create = () => {
+    const setButtonInteractivity = (game, setting) => {
+        const screen = game.state.states[game.state.current];
+        const buttons = screen.scene.getLayouts()[0].buttons;
+
+        fp.map(button => {
+            button.inputEnabled = setting;
+        }, buttons);
+    }
+
+    signal.bus.subscribe({
+        channel: settingsChannel,
+        name: "settings-closed",
+        callback: (args) => {
```

Comment:
We remove brackets when it's just one argument (because you can) so it would be **args =>** rather than **(args) =>** 

Also, _"args"_ is probably a bad name for this (sorry my bad as I used it in that quick code we wrote).

data? Still pretty bad but better than args ...